### PR TITLE
Updated release version to bionic for meilix iso- (Solves -#332)

### LINKED ISSUE #332: Switching to Bionic
Switching in build.sh from xenial to bionic may fail in the debootstrap component.

### PR DESCRIPTION
**Pull request by pranav1698**
### Short description
The release is updated to bionic from xenial
The build passed for my fork,
Link for the releases: https://travis-ci.org/pranav1698/meilix/builds/456026550
The latest release for my fork: https://github.com/pranav1698/meilix/releases/tag/untagged-08b773a3e370def55b2b
  
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

Fixes #332

### CODE REVIEW

**Comment by pranav1698** [comment]
Even though the build is passing and we are getting a released ISO:
I checked the ISO for defects, it was showing defects in two files also the ISO was not working as expected 
I checked for xenial too but their were no defects in that ISO
Even if we change the release to bionic travis is running on xenial, I think that may be the cause of the defects in the iso
Also even after updating the dist to bionic in .tarvis.yml it was switching to precise(maybe it is default) causing the build to because of naturally `package not found` error more in formation can be found here: https://docs.travis-ci.com/user/reference/overview/
So basically travis is not supporting bionic for the image formation
@meets2tarun @tabesin @abishekvashok what do you think?

**Comment by tabesin** [comment]
"I checked the ISO for defects, it was showing defects in two files also the ISO was not working as expected"

What precisely?

**Comment by pranav1698** [comment]
screenshot at 01-06-07
This was the screen that I got it was not happening for xenial

**Comment by pranav1698** [comment]
Even though openbox is installed in iso as default wm, it asks for window manager, after choosing openbox it gives a black screen

**Comment by abishekvashok** [comment]
If you think Travis doesn't support us to go to bionic, then we can containerize things.

**Comment by abishekvashok** [comment]
https://github.com/ricardobranco777/fai

**Comment by tabesin** [comment]
The asking for wm issue is found online as a common request of users. Pls check.

**Comment by pranav1698** [comment]
Thanks for the help @abishekvashok and @tabesin! I will look onto it and soon open a new pr
